namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-interventions.tf

```diff
@@ -0,0 +1,25 @@
+resource "kubernetes_secret" "intervention_events_sns" {
+  metadata {
+    name      = "intervention-events-topic"
+    namespace = "hmpps-interventions-dev"
+  }
+
+  data = {
+    access_key_id     = module.hmpps-domain-events.access_key_id
+    secret_access_key = module.hmpps-domain-events.secret_access_key
+    topic_arn         = module.hmpps-domain-events.topic_arn
+  }
+}
+
+resource "kubernetes_secret" "intervention_global_events_sns" {
```

Comment:
@sldblog why are there two cred pushed into same namespace with same details?  Both secrets will be the same

Reply:
@Mjwillis We're keeping the intervention-events-topic until the application can be reconfigured. I don't want to block work on it at the moment.